addons/src/createNumberMask.js

```diff
@@ -27,6 +27,14 @@ export default function createNumberMask({
   const thousandsSeparatorSymbolLength = thousandsSeparatorSymbol && thousandsSeparatorSymbol.length || 0
 
   function numberMask(rawValue = emptyString) {
+    if(rawValue === '.' && allowDecimal) {
```

Comment:
Can we use `decimalSymbol` here instead.

```js
if(rawValue === decimalSymbol && allowDecimal) {
  let mask = ['0', decimalSymbol, /\d/]
  // ...
```
Then it will still work when someone changes the decimal symbol to something else.